lib/v2.recommended.yaml

```diff
@@ -79,12 +83,20 @@ analyzer:
     avoid_returning_null_for_future: warning
     avoid_returning_null_for_void: warning
     avoid_setters_without_getters: warning
+    avoid_slow_async_io: warning
     avoid_types_on_closure_parameters: warning
     avoid_unused_constructor_parameters: warning
     cascade_invocations: warning
     comment_references: warning
+    control_flow_in_finally: warning
     curly_braces_in_flow_control_structures: warning
+    directives_ordering: warning
+    duplicate_ignore: warning
+    invalid_required_named_param: warning
```

Comment:
Example: `void doThing({@required bool parameter = false});`

Reply:
A few of these didn't have documentation, these are examples that we worked out to show the lints.